README.md

```diff
@@ -1,47 +1,74 @@
 
-Britecharts-react proposes a way of integrating Britecharts and React that promotes a *clear separation of concerns* between the libraries and aims for an *easy and effortless maintenance* of the charts.
-
-## Approach
-The React and D3 integration can be done at different levels, leaning more one side or another. We don’t want to get too specific on the details, as for that there is [slides][d3-react] and [a recording][d3-react-video] that explain the different approaches.
+## Usage
+Britecharts-React components are used just like any other stateless React component. You will pass in some props and it would render a chart:
 
-For this project, we have chosen the approach called ‘Mapping Lifecycle methods’ based on [Nicholas Hery's article][integration-article]. You can see some [code example here][integration-article-code]. 
+```
+<TooltipComponent>
+    <StackedAreaComponent
+        data={stackedAreaData.with2Sources()}
+        width={600}
+        height={400}
+    />
+</TooltipComponent>
 
-This way of integrating react and D3 creates a lightweight React component wrapper that maps React's lifecycle methods `componentDidMount`, `componentDidUpdate` and `componentWillUnmount` into a d3 chart's create, update and unmount methods respectively.
+```
 
-This way, we will end up with a React component that has the usual react dependencies and imports a d3 chart file. This chart module usually contains the d3 dependencies and which, in this case, will contain Britecharts code.
+## API
+Each component's API will be a reflect of [Britecharts][britecharts] charts and their APIs. That way, if we need to render a bar chart, we will first check the [bar chart's API][barChartAPI] in the main project API reference page. You can read more about the approach [here][topics]
 
-## Implementation
-Our initial idea is to create a prototype of this approach using a simple Britecharts chart, iterate over it until we feel confident and only then go ahead and implement the rest of Britecharts.
+From there, we will proceed to pass each of the configurations through the usual props as we do in React projects:
+```
+    <BarComponent
+        data={...}
+        width={400}
+        isHorizontal={true}
+        margin={marginObject}
+    >
 
-We have identified some challenges that we would be looking at in our prototype:
-* The way to move props from the React wrapper to the D3 Chart
-* How to keep ‘DRY' the propTypes, maybe using constants files
-* Figure out the logic to do smart updates, triggering React updates only when a change on the data or configuration happens.
+```
 
-## Usage
-The resulting charts would be used just like any other stateless React component is used. We will pass in some props and it would render a consistent chart.
+## Installation
+Britecharts-React will be available as NPM modules or CDN links.
```

Comment:
Nit: Britecharts-React will be available as an NPM module or through CDN links.

Reply:
better!